doc/theory/glossary.tex

```diff
@@ -307,72 +304,66 @@
-	\end{equation*}
-	where $\lambda_k$ is referred to as the $k$\textit{-layer of the persistence landscape}.
+	\subsection*{Persistence landscape} \label{persistence_landscape}	
 	
 	Let $\{(b_i, d_i)\}_{i \in I}$ be a 
-	% \hyperref[persistence_diagram]{persistence diagram}
-	persistence diagram. Its \textit{associated persistence landscape} $\lambda$ is defined by letting $\lambda_k$ be the $k$-th largest value of the set $\{\Lambda_i(t)\}_ {i \in I}$ where
-	\begin{equation*}
-	\Lambda_i(t) = \left[ \min \{t-b_i, d_i-t\}\right]_+
-	\end{equation*}
-	and $c_+ := \max(c,0)$.
-	
-	Intuitively, we can describe the set of graphs of a persistence landscape by first joining each of the points in the multiset to the diagonal via a horizontal as well as a vertical line, then clockwise rotating the figure 45 degrees and rescaling it by $1/\sqrt{2}$.
-	
-	\paragraph{\\ Reference:} \cite{bubenik2015statistical}
-	
-	\subsection*{Persistence landscape norm} \label{persistence_landscape_norm}
-	
-	Given a function $f : \mathbb R \to \overline{\mathbb R}$ define
-	\begin{equation*}
-	||f||_p = \left( \int_{\mathbb R} f^p(x)\, dx \right)^{1/p}
-	\end{equation*}
-	whenever the right hand side exists and is finite.
-	
-	The $p$\textit{-norm} of a 
-	% \hyperref[persistence_landscape]{persistence landscape}
-	persistence landscape $\lambda = \{\lambda_k\}_{k \in \mathbb N}$ is defined to be
-	
-	\begin{equation*}
-	||\lambda||_p = \left( \sum_{i \in \mathbb N} ||\lambda_i||^p_p \right)^{1/p}
-	\end{equation*}
-	whenever the right hand side exists and is finite.
+	%\hyperref[persistence diagram] {persistence diagram}
+	persistence diagram. Its \textit{persistence landscape} is the set $\{\lambda_k\}_{k \in \mathbb N}$ of functions	
+	\begin{equation*}	
+	\lambda_k : \mathbb R \to \overline{\mathbb R}	
+	\end{equation*}	
+	defined by letting $\lambda_k$ be the $k$-th largest value of the set $\{\Lambda_i(t)\}_ {i \in I}$ where	
+	\begin{equation*}	
+	\Lambda_i(t) = \left[ \min \{t-b_i, d_i-t\}\right]_+	
+	\end{equation*}	
+	and $c_+ := \max(c,0)$. The function $\lambda_k$ is referred to as the \textit{$k$-layer of the persistence landscape}.	
+	
+	Intuitively, we can describe the set of graphs of a persistence landscape by first joining each of the points in the multiset to the diagonal via a horizontal as well as a vertical line, then clockwise rotating the figure 45 degrees and rescaling it by $1/\sqrt{2}$.	
+	
+	The persistence landscape construction defines a
+	%\hyperref[vectorization, kernel and amplitude]{vectorization}
+	vectorization of the set of persistence diagrams with target the vector space of real-valued function on $\mathbb N \times \mathbb R$. For any $p = 1,\dots,\infty$ we can restrict attention to persistence diagrams $D$ whose associated persistence landscape $\lambda$ is
+	%\hyperref[l^p norm]{$p$-integrable}
+	$p$-integrable, that is to say,	
+	\begin{equation} \label{equation: persistence landscape norm}	
```

Comment:
Would mind if we called it `equation:persistence_landscape_norm` instead?

Reply:
OK